contracts/interfaces/IRelayHub.sol

```diff
@@ -256,5 +236,9 @@ interface IRelayHub {
     function penalizeIllegalTransaction(bytes calldata unsignedTx, bytes calldata signature) external;
 
     function getNonce(address from) external view returns (uint256);
+
+    function getHubOverhead() external view returns (uint256);
```

Comment:
?

Reply:
The 'canRelay' now takes parameter 'maxPossibleGas', and it depends on the overhead of the hub. As we plan for upgradeability, the overhead should be exposed by the Hub.